torch/_dynamo/variables/misc.py

```diff
@@ -285,22 +380,34 @@ def call_method(
         args: "List[VariableTracker]",
         kwargs: "Dict[str, VariableTracker]",
     ) -> "VariableTracker":
-        if name != "save_for_backward":
+        if name not in ("saved_tensors", "save_for_backward"):
             unimplemented(f"autograd.Function context method: {name}")
 
         if not self.inference:
             assert self.source and not kwargs
             tx.output.side_effects.track_save_for_backward(self, args)
 
-        return variables.ConstantVariable(
-            None, **VariableTracker.propagate(self, args, kwargs.values())
-        )
+        options = VariableTracker.propagate(self, args, kwargs.values())
+        if name == "saved_tensors":
+            return variables.TupleVariable(list(self._saved_tensors), **options)
+
+        assert name == "save_for_backward"
+        if not hasattr(self, "_saved_tensors"):
+            self._saved_tensors = []
+        for arg in args:
+            # as_proxy can return constant values or other non proxy values
+            if isinstance(arg.as_proxy(), torch.fx.Proxy):
+                arg.as_proxy().node.meta["whitelisted"] = True
+            self._saved_tensors.append(arg)
+        return variables.ConstantVariable(None, **options)
 
     def var_getattr(self, tx, name):
         if name == "save_for_backward":
             return LambdaVariable(
                 lambda *args, **kwargs: self.call_method(tx, name, args, kwargs)
             ).add_options(self)
+        if name == "saved_tensors":
+            return self.call_method(tx, name, [], {})
```

Comment:
nit: Ack, morally `saved_tensors` isn't a method on the ctx object (it's an attribute, right?) Is there a way around this, or is the claim that attributes are just special cases of methods?

Reply:
This just uses the call_method logic for saved_tensors, I suppose I could move this down.